code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -72,7 +72,11 @@
 /obj/item/weapon/robot_module/mommi/soviet/New(var/mob/living/silicon/robot/R) //Powercreep!
 	..()
 
-
+	var/atom/A = locate(/obj/item/device/material_synth/robot/mommi) in src
```

Comment:
AAAAAAAAAAAAAAAAAAAAAAAAAAA

I was supposed to fix this kind of spaghetti by now but you could just remove the default synth for them default mommi module and add it to the NT child

Reply:
Fix the spaghet!